packages/twenty-server/src/engine/core-modules/workspace/types/workspace.type.ts

```diff
@@ -1 +1,8 @@
+import { Workspace } from 'src/engine/core-modules/workspace/workspace.entity';
+
 export type WorkspaceAuthProvider = 'google' | 'microsoft' | 'password' | 'sso';
+
+export type WorkspaceUrlBuilderComponentsType = Pick<
```

Comment:
I don't understand this naming here, I feel it should not be part of the workspace.type.ts but is tied to a "domain related" implementation (like domain-manager for example)

Reply:
maybe move it do domain-manager types instead?